src/app/file-manager/login/login.component.ts

```diff
@@ -35,6 +36,7 @@ export class LoginComponent implements OnInit {
     this.loginService.googleLogin().then(
 
       registered => {
+        AppComponent.updateUserEmail.next(true);
```

Comment:
Why not pass `registered.userInfo.userEmail` here? I think if you do that you won't need to store it in local storage.

Reply:
You are right, I've modified it. It's ready to review, please let me know if you have any other feedback!